lib/connection.js

```diff
@@ -53,18 +52,15 @@ class Connection extends EventEmitter {
     this.application = null;
     this.remoteProxies = {};
 
-    this._heartbeatCallbackInstance = null;
+    this._heartbeatInterval = null;
```

Comment:
I think it will be more appropriate to call this field `_heartbeatTimer` or `_heartbeatTimeout` because it is somewhat confusing to use `interval` here, due to the same word being used as a parameter name in `startHeartbeat` method and having the meaning of the delay between heartbeats.